content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-upgrade-phase.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Fase de atualização do kubeadm
+weight: 90
+content_type: conceito
+---
+
+Na versão v1.15.0, o kubeadm introduziu um suporte preliminar para as fases `kubeadm upgrade node`. Fases para outros subcomandos `kubeadm upgrade`, tal como `apply`, podem ser adicionadas nas seguintes versões.
+
+## Fase de atualização do nó do kubeadm {#cmd-node-phase}
+
+Usando essa fase, você pode optar por executar as etapas separadas da atualização do plano de controle secundário ou nós. Observe que `kubeadm upgrade apply` ainda precisa ser chamado em um nó principal do plano de controle.
+
+{{< tabs name="tab-phase" >}}
+{{< tab name="phase" include="generated/kubeadm_upgrade_node_phase.md" />}}
+{{< tab name="preflight" include="generated/kubeadm_upgrade_node_phase_preflight.md" />}}
+{{< tab name="control-plane" include="generated/kubeadm_upgrade_node_phase_control-plane.md" />}}
+{{< tab name="kubelet-config" include="generated/kubeadm_upgrade_node_phase_kubelet-config.md" />}}
```

Comment:
You should copy `content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase.md` to `content/pt-br/`, it caused build fail.

Reply:
Thank you. /unhold